js/utils.js

```diff
@@ -1588,6 +1630,7 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
       getFields: getRecordFields,
       getFieldValues: getRecordFieldValues,
       parseFilters: parseRecordFilters,
+      setSource: setSourceValue,
```

Comment:
Use a function that takes the LFD configuration as input and then based on the configuration, loads all the necessary values.

```suggestion
      setFilterValues: setFilterValues,
```